wireapp/wire-android
# refactor: migrate UI to material3 [AR-3442]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-3442" title="AR-3442" target="_blank"><img alt="Task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10818?size=medium" />AR-3442</a>  Migrate to material3
  </td></table>
  <br />
 






----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

We are still mixing material2 and material3 components, it appeared recently on the bottom sheet in dark mode with texts being dark on dark background and it's because we use material2 bottom sheet but in its content we use material3 texts.

### Solutions

We should migrate to material3 only, now it should be possible as I checked and almost all material2 components that we still use have already the equivalent in material3 (only `swipeable` is still missing, or rather marked as internal as they are still making it stable enough).

Sorry about the size of it, but mainly it's changing the imports and "experimental" annotations. The elements that have actually been modified are the bottom sheets as they work in a different way in material3.

### Testing

#### How to Test

Open the app in light and dark theme and check if it looks like on the designs.

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
